packages/@aws-cdk/core/lib/stack-synthesizers/default-synthesizer.ts

```diff
@@ -150,6 +150,14 @@ export interface DefaultStackSynthesizerProps {
    * @default true
    */
   readonly generateBootstrapVersionRule?: boolean;
+
+  /**
+   * Prefix to use while storing S3 Assets
+   *
+   *
+   * @default - DefaultStackSynthesizer.DEFAULT_FILE_ASSET_PREFIX
+   */
+  readonly prefix?: string;
```

Comment:
Since this only applies to the S3 file assets, call it something like `bucketPrefix`?